tests/lib/Layout/Resolver/ConditionType/ResourceTypeTest.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Layouts\Sylius\Tests\Layout\Resolver\ConditionType;
+
+use Netgen\Layouts\Sylius\Layout\Resolver\ConditionType\ResourceType;
+use Netgen\Layouts\Sylius\Tests\Stubs\Product;
+use Netgen\Layouts\Sylius\Tests\Stubs\Taxon;
+use Netgen\Layouts\Sylius\Tests\Validator\SettingsValidatorFactory;
+use PHPUnit\Framework\Attributes\CoversClass;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Validator\Exception\UnexpectedTypeException;
+use Symfony\Component\Validator\Validation;
+use Symfony\Component\Validator\Validator\ValidatorInterface;
+
+#[CoversClass(ResourceType::class)]
+final class ResourceTypeTest extends TestCase
+{
+    private ResourceType $conditionType;
+
+    private ValidatorInterface $validator;
+
+    protected function setUp(): void
+    {
+        $allowedResources = [
+            'Sylius\Component\Product\Model\ProductInterface' => 'product',
```

Comment:
Why not use `ProductInterface::class` and `TaxonInterface::class` here?

Reply:
@emodric Fixed in: https://github.com/netgen-layouts/layouts-sylius/pull/14/commits/78c11112af30a4e97f7baffde7382c8f5c593ecd